template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": {
+    "text": "Web Application (Razor Pages)",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
```

Comment:
Do I need to throw in any random GUID or is that something @phenning does when he looks at this file?